rke/k8s_rke_system_images.go

```diff
@@ -2651,6 +2651,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.4"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled out of band for Rancher v2.4.4+
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		"v1.16.14-rancher1-1": {
+			Etcd:                      m("rancher/coreos-etcd:v3.3.15-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.16.14-rancher1"),
+			Alpine:                    m("rancher/rke-tools:v0.1.60"),
```

Comment:
`v1.16.13-rancher1-2` had `rke-tools:v0.1.64`.. was it intentional to change this to an earlier version?

Reply:
No, not sure how that happened. I'll get that updated, and verify the other ones.

Reply:
Cool, looks like the other ones have the same issue